security-framework/src/trust.rs

```diff
@@ -105,6 +106,29 @@ impl SecTrust {
             Ok(TrustResult::from_raw(result))
         }
     }
+
+    /// Returns the number of certificates in an evaluated certificate chain.
+    ///
+    /// Note: evaluate must first be called on the SecTrust.
+    pub fn certificate_count(&self) -> i64 {
```

Comment:
`i64` isn't going to be the right type on 32 bit, right? It probably makes sense to just return CFIndex directly.